apps/web/pages/new-booker/d/[link]/[slug].tsx

```diff
@@ -26,7 +26,7 @@ export default function Type({ slug, user, booking, away, isBrandingHidden, isTe
       <Booker
         username={user}
         eventSlug={slug}
-        rescheduleBooking={booking}
+        bookingData={booking}
```

Comment:
Renaming happened here because we can have bookingData but it's not exclusive to a reschedule.